docs/index.bs

```diff
@@ -3032,6 +3032,9 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
                   1. Wait until |e|'s [=FetchEvent/wait to respond flag=] is unset.
                   1. If |e|'s [=FetchEvent/respond-with error flag=] is set, set |handleFetchFailed| to true.
                   1. Else, set |response| to |e|'s [=FetchEvent/potential response=].
+              1. If |response| is null, |request|'s [=request/body=] is not null, and |request|'s [=request/body=]'s [=Body/source=] is null, then:
+	          1. If |request|'s [=request/body=] is [=Body/unusable=], set |handleFetchFailed| to true.
+		  2. Else, [=ReadableStream/cancel=] |request|'s [=request/body=] with undefined.
```

Comment:
The steps are numbered incorrectly, see screenshot:
![image](https://user-images.githubusercontent.com/649348/108218927-78deba00-7135-11eb-85d4-677a75a7fca2.png)

I think it's because the indentation is inconsistent (tabs mixed with spaces). Also, all numbered steps should be prefixed with `1.`, we shouldn't have a `2.` in there.
```suggestion
              1. If |response| is null, |request|'s [=request/body=] is not null, and |request|'s [=request/body=]'s [=Body/source=] is null, then:
                  1. If |request|'s [=request/body=] is [=Body/unusable=], set |handleFetchFailed| to true.
                  1. Else, [=ReadableStream/cancel=] |request|'s [=request/body=] with undefined.
```

Reply:
Thank you!